### PR TITLE
Subclass the DROLSKY bundle instead of copying most of it

### PR DESCRIPTION
**Pull request by autarch**

### CODE REVIEW

**Comment by autarch** [comment]
Here's a diff of the GeoIP2 perl distro built with the old and new versions of the bundle.

[old-vs-new.diff.txt](https://github.com/maxmind/Dist-Zilla-PluginBundle-MAXMIND/files/467983/old-vs-new.diff.txt)

Some things to note:
- All of the release tests are now distributed in `xt`, rather than being copied into `t`. The diff doesn't follow this and thinks that some files were deleted and new ones added.
- The `perlcriticrc` changes ...
  - I just released a new `Perl::Critic::Moose` that includes `MooseX::Role::Parameterized` in the `Moose::RequireCleanNamespace` policy by default.
  - The naming convention thing that was removed comes from our internal rc file, and is not relevant for our client libraries.
  - Same with the removed `Subroutines::ProhibitManyArgs` policy
  - Some policies were just moved around
- Apparently our old bundle somehow set Perl 5.10 as the min version for the distro, even though the distro itself requires 5.8 (but yet we don't test 5.8 in travis) - I suspect the GeoIP2-perl distro should be fixed here.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+If you use cpanminus, you can do it without downloading the tarball first:
+
+    $ cpanm --reinstall --installdeps --with-recommends Dist::Zilla::PluginBundle::MAXMIND
+
+Dist::Zilla is a very powerful authoring tool, but requires a number of
+author-specific plugins. If you would like to use it for contributing, install
+it from CPAN, then run one of the following commands, depending on your CPAN
+client:
+
+    $ cpan `dzil authordeps --missing`
```
Comment (oalders) [inline CONTRIBUTING.md:50]: These commands are toggling between `cpanm` and `cpan`.  Is this intentional?

Comment (oalders) [inline CONTRIBUTING.md:50]: Ah, I see that it is.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+    $ cpanm Code::TidyAll
+    $ tidyall -a
+
+Please run this before committing your changes and address any issues it
+brings up.
+
+## Contributor Names
+
+If you send me a patch or pull request, your name and email address will be
```
Comment (oalders) [inline CONTRIBUTING.md:119]: Do we care about "us" vs "me" here?

Comment (autarch) [reply CONTRIBUTING.md:119]: I fixed that in my bundle, since it's easy to just remove the word entirely.

**Pull request merged**